src/app/home/home.component.ts

```diff
@@ -1,20 +1,48 @@
 import { Component, OnInit } from '@angular/core';
+import { UsersService, keyCloakService } from '../modules/admin-core';
+import { MatDialog } from '@angular/material';
+import { UnauthorizedComponent } from '../modules/admin-shared';
+
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss']
 })
 export class HomeComponent implements OnInit {
+  promiseRowData: any;
+  admin: boolean = false;
+  constructor(private usersService: UsersService, public dialog: MatDialog,
+    private KeycloakService: keyCloakService, ) { }
 
-  constructor() { }
-
-  ngOnInit() {
+  async ngOnInit() {
+    this.promiseRowData = await this.usersService.getUserRoles();
```

Comment:
Why are we calling the same thing in app component also??

Reply:
No need but it is an initial component , it may be usefull in future i kept, now i am removing